.gitignore

```diff
@@ -11,3 +11,6 @@ pkged.go
 
 node_modules
 /dist
+
+
+.idea/
```

Comment:
I would prefer adding this line to a [global gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) rather than keeping it in the repo. It will be hard for us to keep up with all of the possible IDEs and other configurations folks might have on their local machines